hack/build-image

```diff
@@ -48,6 +48,8 @@ import sys
 
 logger = logging.getLogger("build-image")
 
+FORCE_ARCH_FLAG = False
```

Comment:
What's the real intention here if `FORCE_ARCH_FLAG` is unchanged? Is that some kind of placeholder in case the issue with `--arch` and podman is resolved?

Reply:
In that case should we add a comment here? Otherwise first impression would be similar to mine where it seems kind of useless.

Reply:
Done.